### DISCUSSION TITLE
Resouces.load 뭐가 다른걸까요?

### DISCUSSION THREAD

**Discussion by hwisulee** [naverz/zepeto-studio-kor#1246] [5. Q&A]
![image](https://user-images.githubusercontent.com/62528282/188044450-3560a3b1-802d-4042-acdb-1cd5ccdf077b.png)

위 코드는 리더보드와 연계하여 1~3등 이라면 보여주는 이미지(금,은,동 뱃지)를 바꿔주기 위한 부분입니다.
헌데, 리소스를 불러온 뒤, width, height에서 null 문제가 발생합니다.

![image](https://user-images.githubusercontent.com/62528282/188044608-a67a1d87-f0d1-4302-835d-efeb9d9edd8f.png)

console.log 내부와 주석으로 달은 저 방식을 사용해 리소스를 불러오면 오류가 발생하며 변경되지 않지만, 하드스크립트를 하게 되면 제대로 변경되는 모습을 확인할 수 있었습니다.

`폴더/${변수}`, `"폴더/" + 변수`
위 두 방식과 "폴더/변수"의 차이점이 있을까요?

뭐가 문제일지 모르겠네요...

**Comment by hwisulee** [comment]
방금 오류 코드 찍으려고 다시 실행해봤는데 
![image](https://user-images.githubusercontent.com/62528282/188045258-9b53fe97-e968-4dbb-af52-075a2b8eaaf2.png)

다음과 같은 코드로 제대로 실행 되었습니다...
이해를 할 수가 없네요...

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 말씀하신 내용 확인해 보겠습니다.

**Reply by Zepeto-TechEvangelist** [reply]
확인 결과 어느 방식을 사용해도 동일한 결과가 나오는 것이 정상이며 특별히 내부적으로 다르게 처리하는 부분은 없습니다.
올려주신 예제 코드에서 에셋 이름에 공백이 없는 것을 추천 드립니다.

**Comment by AngeryOrang** [comment]
분명히 플레이 가능하고 테스트를 제대로 하던 코드임에도 다음날이 되면 오류가 나면서 플레이가 안 되는 문제가 부기지수입니다.
코드에 문제가 없는데도 동작이 안 되는데 저도 이걸로 뭔가를 만들어 볼 수 있을지를 모르겠네요.

작업자가 어떤 조치를 하는지 어떤 방식으로 작업하는지에 상관없이 동작을 하던 에러가 나던 이정도면 그냥 랜덤으로 문제가 발생하는 수준입니다.

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 문제 해결 및 개선 작업을 위해 혹시 어떤 부분에서 그런 문제가 발생하는지 상세히 알려주실 수 있으실까요?
